api/app/signals/apps/history/services/signal_log.py

```diff
@@ -203,3 +205,55 @@ def log_receive_feedback(feedback: Feedback) -> None:
             created_at=feedback.submitted_at,
             _signal=feedback._signal,
         )
+
+    @staticmethod
+    def log_external_reaction_received(session: Session, reaction: str) -> None:
+        if not settings.FEATURE_FLAGS.get('SIGNAL_HISTORY_LOG_ENABLED', False):
+            return
+
+        if not isinstance(session, Session):
+            return
+
+        if session.questionnaire.flow != Questionnaire.FORWARD_TO_EXTERNAL:
+            return
+
+        if not session.frozen:
+            return  # log nothing
+
+        external_user = session._signal_status.email_override
+        when = session._signal_status.created_at.strftime('%d-%m-%Y %H:%M')
+        description = f'Toelichting door behandelaar {external_user} op vraag van {when} {reaction}'
+
+        session.history_log.create(
+            action=Log.ACTION_RECEIVE,
+            description=description,
+            created_by=None,
+            created_at=timezone.now(),
+            _signal=session._signal,
+        )
+
+    @staticmethod
+    def log_external_reaction_not_received(session: Session) -> None:
+        if not settings.FEATURE_FLAGS.get('SIGNAL_HISTORY_LOG_ENABLED', False):
+            return
+
+        if not isinstance(session, Session):
+            return
+
+        if session.questionnaire.flow != Questionnaire.FORWARD_TO_EXTERNAL:
+            return
+
+        # if session.frozen or not session.invalidated:
```

Comment:
Is this check commented for a reason? If not used please remove the commented code

Reply:
This is a mistake - will check/fix. (Not problematic because this function is only called in appropriate circumstances.)